os/booting-on-vmware.md

```diff
@@ -45,11 +63,20 @@ The last step uploads the files to your ESXi datastore and registers your VM.
 You can now tweak the VM settings, like memory and virtual cores, then power it
 on. These instructions were tested to deploy to an ESXi 5.5 host.
 
+### Booting with VMware Workstation 12
+
+Run VMware Workstation GUI:
```

Comment:
I don't have these products. if someone provide screenshots - we can add them. But we have to note that we have to update them from time to time on every GUI change.